docker/no-driver/Dockerfile

```
@@ -15,7 +15,7 @@ RUN curl -L -o falco.tar.gz \
     tar -xvf falco.tar.gz && \
     rm -f falco.tar.gz && \
     mv falco-${FALCO_VERSION}-$(uname -m) falco && \
-    rm -rf /falco/usr/src/falco-* /falco/usr/bin/falco-driver-loader
+    rm -rf /falco/usr/src/falco-* /falco/usr/bin/falcoctl
```

Not even sure why is this needed, still...

---

```suggestion
    rm -rf /falco/usr/src/falco-*
```

The `no-driver` was intended not to have the building toolchain. That command was to remove driver-related stuff. Thus, we don't need to remove falcoctl here, IMO.